migrations/1683712628_tagList.ts

```diff
@@ -0,0 +1,64 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+  const newItemTypes: Record<string, SimpleSchemaTypes.ItemType> = {};
+
+  console.log("Create new models/block models");
+
+  console.log('Create block model "Tag list" (`tag_list`)');
```

Comment:
We don't want to keep using naming format `Structured Text -` to identify that this is a structured text block?